default.json5

```diff
@@ -45,6 +45,14 @@
         "https://maven.pkg.github.com/hivemq/hivemq-commons",
       ]
     },
+    {
+      "matchPackageNames": [
+        "hivemq-data-hub"
+      ],
+      "registryUrls": [
+        "https://maven.pkg.github.com/hivemq/hivemq-commons",
+      ]
+    },
```

Comment:
```suggestion
```
This rule is wrong, `hivemq-data-hub` does not belong to `hivemq-commons`.
Actually, it is not yet versioned/published at all, so renovate will not be able to update it.

Reply:
My bad, this is definitely wrong. I will remove it. I read the build script wrong. Do we intend to publish data-hub anytime soon?